test/tests/functional/pbs_qsub_opts_args.py

```diff
@@ -177,3 +193,55 @@ def test_qsub_with_option_a(self):
         except PbsSubmitError as e:
             self.validate_error(e)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid_2)
+
+    @requirements(num_moms=2)
+    def test_qsub_sandbox_private_jobdir_shared(self):
+        """
+        Test submission of job with sandbox=PRIVATE,
+        and moms have $jobdir_root set to shared.
+        """
+        self.remove_jobdir = True
+        momA = self.moms.values()[0]
+        momB = self.moms.values()[1]
+
+        c = {'$jobdir_root': '%s shared' % self.jobdir_root}
+        l = {'$logevent': 4095}
+        for mom in [momA, momB]:
+            mom.add_config(c)
+            if mom == momB:
+                mom.add_config(l)
```

Comment:
Move this outside the for and just say momB.add_config()

Reply:
If I do that, then it would look like this:
```
c = {'$jobdir_root': '%s shared' % self.jobdir_root}
for mom in [momA, momB]:
    mom.add_config(c)
    mom.restart()
l = {'$logevent': 4095}
momB.add_config(l)
momB.restart()
```
Or would you prefer this:
```
 c = {'$jobdir_root': '%s shared' % self.jobdir_root}
momA.add_config(c)
momA.restart()
l = {'$logevent': 4095}
momB.add_config(c)
momB.add_config(l)
momB.restart()
```

Reply:
The first restarts momB twice.  This is unnecessary.  If you add 'l' first, you can then do your for loop and just restart the mom there.  It is going to be the shortest.  The second is also fine.  Whichever you want.  Just don't restart momB twice.  As a note, pycodestyle will complain at you for the use of a variable called 'l'.  It considers it confusing.  You might consider renaming it.

Reply:
I'll rename the variable.